btr-api/src/btr_api/utils/redact.py

```diff
@@ -1,7 +1,6 @@
 """Function to redact fields and or a submission based on roles with a variety of options"""
 from flask import current_app
 
-from btr_api.services import btr_auth
```

Comment:
needed to remove this to prevent a cyclic import when other 'services' import anything from utils